src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.ProjectState.cs

```diff
@@ -223,6 +224,8 @@ public bool IsEmpty(DocumentId documentId)
                 }
 
                 await SerializeAsync(serializer, project, result.ProjectId, _owner.NonLocalStateName, result.Others).ConfigureAwait(false);
+
+                AnalyzerABTestLogger.LogProjectDiagnostics(project, result);
```

Comment:
FYI. when FSA is off, we never collect diagnostics for whole project. so we can't log error count for whole project is FSA is off. and this will never log anything for FSA off. only document one, the one below this, will log count for FSA off.